CMR/python/test/cmr/auth/test_token.py

```diff
@@ -376,7 +376,7 @@ def test_fetch_token_delete_path(self, now_mock, deltoken_mock, readtoken_mock,
         user = 'tester'
         config = {'cmr.token.value':'pass'}
 
-        now_mock.return_value = datetime.strptime('12/28/2021 09:13:20', '%m/%d/%Y %H:%M:%S')
+        now_mock.return_value = datetime.strptime('12/28/2021 09:13:20 EST', '%m/%d/%Y %H:%M:%S %Z')
```

Comment:
Why did you set the time zone to EST? User could be on PST.

Reply:
in the test it does not matter, you just need to pick a timezone, this line of code actually freezes time, I'm mocking the now function to always return one specific point in time.